_rules/SC2-4-6-descriptive-labels.md

```diff
@@ -148,23 +199,58 @@ Label is included in accessibility tree, but not visible, and does not describe
 <input aria-labelledby="label_fname" type="text" name="fname"/>
 ```
 
+#### Failed Example 6
+
+Accessible name created through "aria-label" does not describes the purpose of the associated element.
+
+```html
+<input aria-label="Menu" id="fname" type="text" name="fname"/>
+```
+
+#### Failed Example 7
+
+The `label` in itself does not describe the purpose of the form field, and while the context can be relied upon to differentiate form fields from each other, it is not sufficient to rely on the context for describing the entire purpose of the form field.
+
+```html
+<h2>Name</h2>
+<label>Fill in: <input id="name" type="text" name="name"/></label>
+````
+
+#### Failed Example 8
+
+The descriptive label coded with the `label` element is through the accessible name calculation overwritten by an `aria-label` that is not descriptive.
```

Comment:
Perhaps we could add here something that makes it clearer why this fails. Something along these lines:
While Expectation 1 is met, Expectation 2 is not, and no expectation can fail.